Makefile

```diff
@@ -33,4 +34,28 @@ $(ROMS_DIR)/%: roms.zip
 	unzip $< 'roms/*' -x '*/.DS_Store'
 	touch roms/*
 
-.PHONY: requirement roms
+CURRENT_OS=
+ifeq ($(OS),Windows_NT)
+	CURRENT_OS := windows
+else
+	UNAME_S := $(shell uname -s)
+	ifeq ($(UNAME_S),Linux)
+		CURRENT_OS := Linux
+	endif
+	ifeq ($(UNAME_S),Darwin)
+		CURRENT_OS := MacOS
+	endif
+endif
+
+extern_dependencies:
+ifeq ($(CURRENT_OS),Linux)
+	sudo apt install cmake libgtk-3-dev
```

Comment:
not all linux deps have apt ;)

don't bother with deps in makefile,
just list them in the `README`